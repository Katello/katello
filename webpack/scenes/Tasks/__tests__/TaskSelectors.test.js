import { testSelectorsSnapshotWithFixtures } from 'react-redux-test-utils';
import {
  selectIsPollingTask,
  selectIsPollingTasks,
} from '../TaskSelectors';

const state = {
  intervals: {
    TEST_POLL_TASK: 1,
    TEST_TASK_SEARCH: 3,
  },
};

const fixtures = {
  'selects if polling a task': () => selectIsPollingTask(state, 'TEST'),
  'selects if not polling a task': () => selectIsPollingTask({}),
  'selects if polling tasks': () => selectIsPollingTasks(state, 'TEST'),
  'selects if not polling tasks': () => selectIsPollingTasks({}),
};

describe('Task selectors', () => testSelectorsSnapshotWithFixtures(fixtures));
