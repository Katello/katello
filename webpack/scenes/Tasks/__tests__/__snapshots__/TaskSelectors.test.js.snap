// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Task selectors selects if not polling a task 1`] = `false`;

exports[`Task selectors selects if not polling tasks 1`] = `false`;

exports[`Task selectors selects if polling a task 1`] = `true`;

exports[`Task selectors selects if polling tasks 1`] = `true`;
